README.md

```diff
@@ -78,19 +78,21 @@ Please see the dedicated [README.md](tools/docker/README.md) for information on
 * [Concepts](doc/conceptual-guide.md)
 * [Iceoryx Hoofs Hacker Guide](iceoryx_hoofs/README.md)
 
-### Targeted quality levels & platforms
+### Quality levels & platforms
 
 > [Quality level](./CONTRIBUTING.md#quality-levels) are 5 to 1+, where 1+ is highest level.
 
-|CMake project/target                     | QNX  | Linux, Windows, MacOS | Comment                             |
-|-----------------------------------------|:----:|:---------------------:|:-----------------------------------:|
-| iceoryx_hoofs                           | 1+   | 2                     |                                     |
-| iceoryx_posh                            | 1+, 2| 2                     | Will be split into separate targets |
-| iceoryx_binding_c                       | 2    | 2                     |                                     |
-| iceoryx_examples                        | 5    | 5                     | All example code in this folder     |
-| iceoryx_dds                             | 4    | 4                     |                                     |
-| iceoryx_introspection                   | 5    | 5                     |                                     |
-| iceoryx_meta                            | 5    | 5                     |                                     |
+Please see the [Quality Declaration](./QUALITY_DECLARATION.md) for details of the quality measures according to ROS 2 guidelines.
+
+|CMake project/target   | Current Level | Target Level QNX  | Target Level <br> Linux, Windows, MacOS | Comment                             |
+|-----------------------|:-------------:|:-----------------:|:---------------------------------------:|:-----------------------------------:|
+| iceoryx_hoofs         | 2             | 1+                | 1                                       |                                     |
+| iceoryx_posh          | 2             | 1+, 2             | 1                                       | Will be split into separate targets |
+| iceoryx_binding_c     | 2             | 2                 | 1                                       |                                     |
```

Comment:
I think the Target Level QNX is here also 1+

Reply:
Agreed